content/applications/hr/fleet/models.rst

```diff
@@ -0,0 +1,195 @@
+  configurator for future employees. This value impacts the gross and net salary of the employee
+  assigned to the vehicle. This figure is depreciated over time according to local tax laws. The
+  :guilabel:`Cost (Depreciated)` does not depreciate automatically on the *vehicle model*, it only
+  depreciates based on the *contract* linked to a specific vehicle and not on the general model.
+- :guilabel:`Total Cost (Depreciated)`: this value is the :guilabel:`Cost (Depreciated)` and the
+  :guilabel:`C02 fee` fields combined, and also is depreciated over time.
+
+Engine
+~~~~~~
+
+- :guilabel:`Fuel Type`: select the type of fuel the vehicle uses from the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: enter the average carbon dioxide emissions the vehicle produces in
+  grams per kilometer (g/km). This information is provided by the car manufacturer.
+- :guilabel:`CO2 Standard`: enter the standard amount of carbon dioxide in grams per kilometer
+  (g/km) for a similar sized vehicle.
+- :guilabel:`Transmission`: select :guilabel:`Manual` or :guilabel:`Automatic` transmission type
+  from the drop-down menu.
+- :guilabel:`Power`: if the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Horsepower`: enter the vehicle's horsepower in this field.
+- :guilabel:`Horsepower Taxation`: enter the amount that is taxed based on the size of the vehicle's
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Tax Deduction`: this field auto-populates according to the engine specifications, and
+  cannot be modified. The percentage is based on the localization settings and local tax laws.
+
+Vendors tab
+-----------
+
+Specify the vendors a vehicle can be purchased from in this tab. With proper setup, requests for
+quotations for vehicles can be easily created through Odoo's *Purchase* app.
+
+The vendors the vehicle can be purchased from may be added in this tab. To add a vendor, click
+:guilabel:`Add`, which opens a pop-up window with a list of all the vendors currently in the
+database. Select the vendor to add by clicking the checkbox next to the vendor name, then click
+:guilabel:`Select`. There is no limit to the number of vendors that can be added to this list.
+
+If a vendor is not in the database, add a vendor by clicking :guilabel:`New`. In a new vendor form,
+enter the information in the vendor tab, then click :guilabel:`Save & Close` to add the vendor and
+close the window, or :guilabel:`Save & New` to add the current vendor and create another new vendor.
```

Comment:
```suggestion
If a vendor is not in the database, add a vendor by clicking :guilabel:`New`. 

In the :guilabel:`Create Vendors` form that appears, enter the necessary information, then click :guilabel:`Save & Close` to add the vendor, or click :guilabel:`Save & New` to add the current vendor and create another new vendor.
```
I'm not seeing a 'Vendor' tab...
![Screenshot 2024-06-27 122459](https://github.com/odoo/documentation/assets/73958186/72c37814-69f4-404e-b799-c43bb448d12e)